spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
```

Comment:
This FactoryBot for Users was not used in this current version because of the mock_user method, but we can see how to use it in future versions.